pybop/optimisers/scipy_optimisers.py

```diff
@@ -3,36 +3,63 @@
 
 from .base_optimiser import BaseOptimiser
 
+DEFAULT_SCIPY_MINIMIZE_OPTIONS = dict(
+    bounds=None,
+    method="Nelder-Mead",
+    tol=1e-5,
+    options=dict(),
+)
+DEFAULT_SCIPY_DIFFERENTIAL_EVOLUTION_OPTIONS = dict(
+    bounds=None,
+    strategy="best1bin",
+    popsize=15,
+    maxiter=1000,
```

Comment:
Will this be used? If I understand the call structure correctly, this will always be overwritten from the `optimisation.set_max_iterations()` method.